litestream/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM litestream/litestream:0.3.13
```

Comment:
this image does not have `ytt` or curl

you can do something like this to load `ytt` into the container bin and use it to map env values at run time

https://github.com/ar-io/arns-service/blob/litestream/litestream/Dockerfile

Reply:
fixed